### PR TITLE
Improve prefilled issue template

### PR DESCRIPTION
**Pull request by Thomas1664**
- Fix markdown
- Use `Color::warning` to better distinguish it from the rest of the error message
- Fix printed path: The `.` after the path prevents users from navigating to the issue template directly from within the terminal.

### CODE REVIEW

File: src/vcpkg/commands.version.cpp [hunk 1]
```diff
@@ -9,9 +9,9 @@
 namespace
 {
     using namespace vcpkg;
-    constexpr StringLiteral version_init = VCPKG_BASE_VERSION_AS_STRING "-" VCPKG_VERSION_AS_STRING
+    static constexpr StringLiteral version_init = VCPKG_BASE_VERSION_AS_STRING "-" VCPKG_VERSION_AS_STRING
```
Comment (BillyONeal) [inline src/vcpkg/commands.version.cpp:12]: I think you should revert this change because it has no effect. (entities declared within anonymous namespaces already have internal linkage, see http://eel.is/c++draft/basic.link#4 )

File: include/vcpkg/build.h [hunk 2]
```diff
@@ -220,9 +220,19 @@ namespace vcpkg
...
-                                                        Optional<Path>&& issue_body = nullopt);
+    LocalizedString create_user_troubleshooting_message(const InstallPlanAction& action, const VcpkgPaths& paths);
+    inline void print_user_troubleshooting_message(const InstallPlanAction& action,
+                                                   const VcpkgPaths& paths,
+                                                   Optional<Path>&& issue_body)
+    {
+        msg::println_error(create_user_troubleshooting_message(action, paths));
+        if (issue_body)
+        {
+            // Intentionally not using `println_warning` here to avoid `warning: ` prefix
```
Comment (BillyONeal) [inline include/vcpkg/build.h:231]:
```suggestion
```

I think it's clear without the comment: it's printing a troubleshooting message, not a warning. Could go either way though.

File: include/vcpkg/build.h [hunk 3]
```diff
@@ -220,9 +220,19 @@ namespace vcpkg
 
     StringLiteral to_string_locale_invariant(const BuildResult build_result);
     LocalizedString to_string(const BuildResult build_result);
-    LocalizedString create_user_troubleshooting_message(const InstallPlanAction& action,
-                                                        const VcpkgPaths& paths,
-                                                        Optional<Path>&& issue_body = nullopt);
+    LocalizedString create_user_troubleshooting_message(const InstallPlanAction& action, const VcpkgPaths& paths);
```
Comment (BillyONeal) [inline include/vcpkg/build.h:223]: Since it's now an error to call this IMO it would be better to just inline it (and put the definition of `print_user_troubleshooting_message` into `build.cpp`?)

**Review by BillyONeal — APPROVED** [review]
Only nitpicks here. Thanks for your contribution!

File: include/vcpkg/build.h (hunk 3 above)
Comment (Thomas1664) [reply include/vcpkg/build.h:223]: This is not possible:

```
vcpkg-tool\src\vcpkg\install.cpp(525,17): error C3861: "print_user_troubleshooting_message": identifier not found
```

**Comment by BillyONeal** [comment]
Thank you!
